### PR TITLE
added 2nd and 3rd wishlists 

### PR DESCRIPTION
**Pull request by fvela009**

### CODE REVIEW

**Review by QuickWick123 — APPROVED** [review]

**Review by QuickWick123 — APPROVED** [review]

File: wishlist/urls.py [hunk 1]
```diff
@@ -3,7 +3,13 @@
 
 urlpatterns = [
     path('wishlist/', views.wishlist, name='wishlist'),
-    path('wishlist/d/<int:book_id>', views.deleteItem, name='deleteItem2'),
+    path('wishlist/d/<int:book_id>', views.deleteItem, name='deleteItem'),
```
Comment (Mar-Sua) [inline wishlist/urls.py:6]: I believe this is the line that was conflicting with the cart page due to them both delete buttons having the same name. Please update the name.

Comment (fvela009) [reply wishlist/urls.py]: changed it

**Review by houyhourung — APPROVED** [review]
looks good

**Review by Mar-Sua — APPROVED** [review]
Lots of migrations. Everything else looks good.

**Pull request merged**